tests/integration/ledger/mocks/acc_retriever.go

```diff
@@ -0,0 +1,29 @@
+package mocks
+
+import (
+	"github.com/cosmos/cosmos-sdk/client"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+var _ client.AccountRetriever = (*MockAccountRetriever)(nil)
+
+// MockAccountRetriever defines a no-op basic AccountRetriever that can be used
+// in mocked contexts. Tests or context that need more sophisticated testing
+// state should implement their own mock AccountRetriever.
+type MockAccountRetriever struct{}
+
+func (mar MockAccountRetriever) GetAccount(_ client.Context, _ sdk.AccAddress) (client.Account, error) {
```

Comment:
Why did you take this approach instead of using `testify/mock`?

Reply:
Got 'as is' from cosmos-sdk